src/index.ts

```diff
@@ -267,8 +267,15 @@ function startApp(appSchema, path: string) {
 
 const app = express()
 
-// This order is important for dd-trace to be able to find the nested routes.
-app.use("/v2", startApp(schemaV2, "/"))
-app.use("/", startApp(schemaV1, "/"))
+;(async () => {
+  try {
+    const schemaV2 = await getSchemaV2()
+    // This order is important for dd-trace to be able to find the nested routes.
+    app.use("/v2", startApp(schemaV2, "/"))
+    app.use("/", startApp(schemaV1, "/"))
+  } catch (error) {
+    console.log(error)
+  }
+})()
```

Comment:
This is where the double page refresh is required. Below we're returning `export default app`, which is mounted into the main express app but here we need to wait for `getSchemaV2` to return before the routes are actually mounted -- hence the hiccup.